doc/guides/west/manifest.rst

```diff
@@ -1179,6 +1189,430 @@ processed in this order:
 - If the value is a sequence, its elements are recursively imported in the
   order they appear.
 
+.. _west-manifest-groups:
+
+Project Groups and Active Projects
+**********************************
+
+You can use the ``groups`` keys briefly described :ref:`above
+<west-manifest-files>` to put projects into groups. The ``groups`` key can be a
+project or top level manifest attribute, like this:
+
+.. code-block:: yaml
+
+   manifest:
+     projects:
+       - name: some-project
+         groups: ...
+     groups: ...
```

Comment:
Can you move this above `projects:` so that  the order in the file matches the description above?

Reply:
I don't follow this one. The description above says:

```
The ``groups`` key can be
a project or top level manifest attribute
```

This ordering matches that description.

Reply:
Oh, I misread the "top-level attribute" sentence. You are indeed right.